server/lib/update-fields.js

```diff
@@ -0,0 +1,40 @@
+import Customer from '../models/customer'
+import Volunteer from '../models/volunteer'
+
+export default {
+  /**
+   * Update Volunteer fields of address
+   */
+  async updatingVolunteer(req) {
+    //Updating address of the volunteer role if exists
+    const volunteer = await Volunteer.findById(req.body._id)
+    if(volunteer != undefined && volunteer != null && volunteer != {}){
+      let oldAdrFields = []
```

Comment:
Yeah if you could combine it all in a single function it would be nice. 

I forgot to say updating donors should be fixed now if you pull the latest staging.